svo (refiller).xml

```diff
@@ -315,30 +229,48 @@ echo'\n' svo.echof("Handed all vials over.")</script>
 				<integer>3</integer>
 			</regexCodePropertyList>
 		</Trigger>
+		<Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
+			<name>There's something wrong there</name>
+			<script>svo.rf_cancel()</script>
+			<triggerType>0</triggerType>
+			<conditonLineDelta>0</conditonLineDelta>
+			<mStayOpen>0</mStayOpen>
+			<mCommand></mCommand>
+			<packageName></packageName>
+			<mFgColor>#ff0000</mFgColor>
+			<mBgColor>#ffff00</mBgColor>
+			<mSoundFile></mSoundFile>
+			<colorTriggerFgColor>#000000</colorTriggerFgColor>
+			<colorTriggerBgColor>#000000</colorTriggerBgColor>
+			<regexCodeList>
+				<string>That pot already contains an</string>
+				<string>That alembic already contains a</string>
+				<string>That is not a pot.</string>
+				<string>That is not an alembic.</string>
+			</regexCodeList>
```

Comment:
Changed them to exact matches. Also fixed the first ones by using a single perl regex trigger for both alembic and pot and also considers a and an.

Reply:
I liked it how it was before, two different ones for pot and alembic. Regex is slower than begin-of-line substring slightly, so that would be preferred... 

That said, is this trigger usually enabled or disabled? Looks enabled by default the way you have it right now, so should definitely optimize that. If it is disabled when not brewing, wouldn't be so necessary

Reply:
Alright I disabled it by default and made it enable upon refilling and disable when there is nothing left or cancelled/failed.